### DISCUSSION TITLE
Add links to the documentation when warning in the terminal

### DISCUSSION THREAD

**Discussion by titouanmathis** [tailwindlabs/tailwindcss#2751] [Ideas]
Hi,

It is not always straight forward to understand what to do when a warning is shown upon build, would it be possible to add links to the documentation explaining why the warning exists and how it could be fixed? 

For example, when the following warning is shown: 

```
warn - The `conservative` purge mode will be removed in Tailwind 2.0.                                                                                                                                         
warn - Please switch to the new `layers` mode instead. 
```

We could have a link to the [Purge layers by default](https://tailwindcss.com/docs/upcoming-changes#purge-layers-by-default) documentation:

```
warn - The `conservative` purge mode will be removed in Tailwind 2.0.                                                                                                                                         
warn - Please switch to the new `layers` mode instead. 
warn - Doc: https://tailwindcss.com/docs/upcoming-changes#purge-layers-by-default
```

What are your thoughts on this?

**Comment by damianof** [comment]
I agree with this suggestion. It would help
